app/src/main/res/values/dimens.xml

```diff
@@ -10,8 +10,10 @@
     <dimen name="huge_gap">48dp</dimen>
     <dimen name="large_gap">32dp</dimen>
     <dimen name="standard_gap">16dp</dimen>
+    <dimen name="medium_gap">12dp</dimen>
     <dimen name="small_gap">8dp</dimen>
     <dimen name="tiny_gap">4dp</dimen>
+    <dimen name="gap">2dp</dimen>
```

Comment:
Is it the usual way to call it? Otherwise feel free to call it minuscule_gap for instance :-)

Reply:
I don't know. I called gap because I don't know how sizes are named in English. If you want, I can change the name.

Reply:
@m-alzam please rename it to `minuscule_gap`. That should be fine.

Reply:
Done